website/server/models/group.js

```diff
@@ -366,6 +366,17 @@ schema.methods.startQuest = async function startQuest (user) {
   let nonUserQuestMembers = _.keys(this.quest.members);
   removeFromArray(nonUserQuestMembers, user._id);
 
+  // remove any users from quest.members who aren't in the party
+  let partyId = this._id;
+  let questMembers = this.quest.members;
+  await Bluebird.map(Object.keys(this.quest.members), async (memberId) => {
+    let member = await User.findById(memberId, 'party._id').exec();
```

Comment:
In fact, let's add a test for that case. The case where a user deleted their account and it wasn't cleaned up properly.